src/benchmark/tpch_benchmark.cpp

```diff
@@ -94,7 +94,9 @@ int main(int argc, char* argv[]) {
   std::cout << "- Benchmarking Queries: [ ";
   for (const auto& item_id : item_ids) {
     std::cout << (item_id + 1);
-    if (item_id != item_ids.back()) { std::cout << ","; }
+    if (item_id != item_ids.back()) {
+      std::cout << ",";
```

Comment:
Can't we append `", "` and then `std::cout << " ]" << std::endl;` (if we really want these nasty spaces next to the brackets)?

Reply:
Did you mean to comment on this PR?

Reply:
Yep ... it's not really their change, but if they are going to change for formatting anyways ...

Reply:
I feel like we have already added more than enough to this PR